aws-s3/authorization.ml

```diff
@@ -103,7 +103,7 @@ let make_auth_header ~credentials ~scope ~signed_headers ~signature =
     signed_headers
     signature
 
-let make_presigned_url ?(scheme=`Https) ?host ?port ~credentials ~date ~region ~path ~bucket ~verb ~duration () =
+let make_presigned_url ?(scheme=`Https) ?host ?port ?(custom_query=[]) ~credentials ~date ~region ~path ~bucket ~verb ~duration () =
```

Comment:
As `custom_query` argument a list of additional query parameters, I suggest changing the name to `query` to match the naming convention for the other functions in the module.